RelhaxModpack/RelhaxModpack/Shortcuts/ShortcutUtils.cs

```diff
@@ -106,6 +125,70 @@ internal interface IShellLink
             void Resolve(IntPtr hwnd, int fFlags);
             void SetPath([MarshalAs(UnmanagedType.LPWStr)] string pszFile);
         }
+
+        // needed for CreateShortcut
+        [Flags]
+        public enum StgmConstants
+        {
+            STGM_READ = 0x0,
+            STGM_WRITE = 0x1,
+            STGM_READWRITE = 0x2,
+            STGM_SHARE_DENY_NONE = 0x40,
+            STGM_SHARE_DENY_READ = 0x30,
+            STGM_SHARE_DENY_WRITE = 0x20,
+            STGM_SHARE_EXCLUSIVE = 0x10,
+            STGM_PRIORITY = 0x40000,
+            STGM_CREATE = 0x1000,
+            STGM_CONVERT = 0x20000,
+            STGM_FAILIFTHERE = 0x0,
+            STGM_DIRECT = 0x0,
+            STGM_TRANSACTED = 0x10000,
+            STGM_NOSCRATCH = 0x100000,
+            STGM_NOSNAPSHOT = 0x200000,
+            STGM_SIMPLE = 0x8000000,
+            STGM_DIRECT_SWMR = 0x400000,
+            STGM_DELETEONRELEASE = 0x4000000
+        }
+
+        //https://github.com/Slashka-DK/BlitzChat/blob/d6d5a5029caa925496a4cda85867a541d2f0e7de/packages/WPF%20Shell%20Integration%20Library%20v2/Microsoft.Windows.Shell/Standard/NativeMethods.cs
+        internal static class Win32Value
+        {
+            public const uint MAX_PATH = 260;
+            public const uint INFOTIPSIZE = 1024;
+            public const uint TRUE = 1;
+            public const uint FALSE = 0;
+            public const uint sizeof_WCHAR = 2;
+            public const uint sizeof_CHAR = 1;
+            public const uint sizeof_BOOL = 4;
+        }
+
+        //https://github.com/Slashka-DK/BlitzChat/blob/d6d5a5029caa925496a4cda85867a541d2f0e7de/packages/WPF%20Shell%20Integration%20Library%20v2/Microsoft.Windows.Shell/Standard/NativeMethods.cs
+        [Flags]
+        enum SLGP
```

Comment:
REEEEEEEEEEEEEEEEEEEEEEEEEE x2

Reply:
These are specific to that one class and they're in a `#region` of "gross shortcut stuff". I could justify leaving them there?

Reply:
var justified = true